external/radiation/get_data.sh

```diff
@@ -9,21 +9,11 @@ else
 
     export MYHOME=`pwd`
 
-    if [ ! -d "./data/fortran/radiation_driver)" ]; then
-        mkdir -p ./data/fortran/radiation_driver
-        gsutil -m cp -r gs://vcm-fv3gfs-serialized-regression-data/physics/radiation_driver/* ./data/fortran/radiation_driver
-        cd ./data/fortran/radiation_driver
-        tar -xzvf dat_files.tar.gz
-        cd $MYHOME
-    else
-        echo "Driver standalone data already present"
-    fi
-
     if [ ! -d "./data/lookupdata" ]; then
         cd ./data
         mkdir lookupdata
         cd ./lookupdata
-        gsutil cp -r gs://vcm-fv3gfs-serialized-regression-data/physics/lookupdata/lookup.tar.gz .
+        gsutil cp -r gs://vcm-ml-intermediate/radiation/lookupdata/lookup.tar.gz .
```

Comment:
@brianhenn, I moved the data into intermediate for now

Reply:
Thanks @frodre that sounds good. One further thing is that we don't need get_data.sh at all with the serialbox regression test removed.